release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -190,6 +190,13 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-networking-ipi-nmstate"]
+==== Configuring host network interfaces with NMState on installer-provisioned clusters
+
+{product-title} now provides a `networkConfig` configuration setting for installer-provisioned clusters, which takes an NMState YAML configuration to configure host interfaces. During installer-provisioned installations, you can add the `networkConfig` configuration setting and the NMState YAML configuration to the `install-config.yaml` file. Additionally, you can add the `networkConfig` configuration setting and the NMState YAML configuration to the bare metal host resource when using the Bare Metal Operator.
+
+The most common use case for the `networkConfig` configuration setting is to set static IP addresses on a host's network interface during installation or while expanding the cluster.
```

Comment:
LGTM. Did you want to add a link to the feature?

Reply:
Added hyperlink.